.eslintrc.cjs

```diff
@@ -7,8 +7,35 @@ module.exports = {
     '@electron-toolkit/eslint-config-prettier',
     'plugin:storybook/recommended',
     'plugin:react-hooks/recommended',
+    'plugin:import/recommended',
   ],
   rules: {
     '@typescript-eslint/explicit-function-return-type': 0,
+    'import/no-named-as-default-member': 0,
+    'import/order': 'error',
+    'no-restricted-imports': [
+      'error',
+      {
+        patterns: [
+          '../*', // Disallow relative imports that go up a directory
+          './*', // Optionally disallow relative imports within the same directory
+          '!./*.module.css', // allow relative import of CSS module files
+          'src/*', // Disallow absolute imports starting with `src/` and enforce alias usage
```

Comment:
I am wondering if we specify ts/js as there may be other files that make sense to import directly other than just module.css

Also, is part of the goal with this change (`@engine`, etc) to help with plugins or the web export/sub packages, etc?

Reply:
So I'm going to remove the rule `./*` because I think it's fine to import sibling files. It's just when we start going up and down the file structure with lots of `/../` that it gets a bit unwieldy.

The `@engine` stuff is just for tidiness when importing. For instance rather than having to do `src/renderer/components/whatever` put the whole import path, we can just do `@components/whatever`. It also means we could potentially change the location of that folder and all we'd need to update is the alias path. In the case of `@engine` though this will probably change quite soon when we move it to a monorepo...

Reply:
Ok I see your point now, I'm not going to allow neighbouring directories to be imported but neighbouring files (as you suggested) makes sense.